include/event.php

```diff
@@ -146,63 +152,81 @@ function format_event_bbcode($ev) {
 
 	$o = '';
 
-	if($ev['summary'])
+	if ($ev['summary']) {
 		$o .= '[event-summary]' . $ev['summary'] . '[/event-summary]';
+	}
 
-	if($ev['desc'])
+	if ($ev['desc']) {
 		$o .= '[event-description]' . $ev['desc'] . '[/event-description]';
+	}
 
-	if($ev['start'])
+	if ($ev['start']) {
 		$o .= '[event-start]' . $ev['start'] . '[/event-start]';
+	}
 
-	if(($ev['finish']) && (! $ev['nofinish']))
+	if (($ev['finish']) && (! $ev['nofinish'])) {
 		$o .= '[event-finish]' . $ev['finish'] . '[/event-finish]';
+	}
 
-	if($ev['location'])
+	if ($ev['location']) {
 		$o .= '[event-location]' . $ev['location'] . '[/event-location]';
+	}
 
-	if($ev['adjust'])
+	if ($ev['adjust']) {
 		$o .= '[event-adjust]' . $ev['adjust'] . '[/event-adjust]';
-
+	}
 
 	return $o;
-
 }
 
 function bbtovcal($s) {
 	$o = '';
 	$ev = bbtoevent($s);
-	if($ev['desc'])
+
+	if ($ev['desc']) {
 		$o = format_event_html($ev);
+	}
+
 	return $o;
 }
 
-
 function bbtoevent($s) {
 
 	$ev = array();
 
 	$match = '';
-	if(preg_match("/\[event\-summary\](.*?)\[\/event\-summary\]/is",$s,$match))
+	if (preg_match("/\[event\-summary\](.*?)\[\/event\-summary\]/is",$s,$match)) {
```

Comment:
Standards: Can you please add a space after commas in this `preg_match` and all the subsequent ones?